README.md

```diff
@@ -545,12 +560,28 @@ that the Kotlin code has been improved, but the underlying Java library remained
 
 Pulumi Kotlin SDK is a proof of concept, **we really need feedback before moving on**.
 
-- Create [an issue in this repo](https://github.com/VirtuslabRnD/pulumi-kotlin/issues) or express your opinion in the
-  existing ["Support for idiomatic Kotlin"](https://github.com/pulumi/pulumi-java/issues/544) issue.
-- Start a thread on [#java channel](https://pulumi-community.slack.com/archives/C03DPAY96NB) (Pulumi community Slack).
-- Book a meeting [through Calendly](https://calendly.com/michalfudala/kotlin-sdk-for-pulumi-feedback) and let's talk!
+- Create [an issue in this repo][issues-pulumi-kotlin] or express your opinion in the
+  existing ["Support for idiomatic Kotlin"][support-for-idiomatic-kotlin-issue] issue.
+- Start a thread on [#java channel][pulumi-slack-java-channel] (Pulumi community Slack).
+- Book a meeting [through Calendly][calendly-feedback-meeting] and let's talk!
 
 ## Development docs
 
 - [Code generation and local run details](docs/code-generation.md)
 - [Development guidelines](docs/development-guidelines.md)
+
+
+[github-packages-pulumi-kotlin]: https://github.com/orgs/VirtuslabRnD/packages?repo_name=pulumi-kotlin
+[github-maven-public-repos-discussion]: https://github.com/orgs/community/discussions/25979
+[pulumi-kotlin-code-examples]: https://github.com/VirtuslabRnD/pulumi-kotlin/tree/main/examples
+[pulumi-kotlin-issue-197]: https://github.com/VirtuslabRnD/pulumi-kotlin/issues/197
+[pulumi-installation-guide]: https://www.pulumi.com/docs/reference/install/?utm_campaign=pulumi-pulumi-github-repo&utm_source=github.com&utm_medium=getting-started-install
+[github-new-access-token]: https://github.com/settings/tokens/new
+[google-native-kdoc]: https://storage.googleapis.com/pulumi-kotlin-docs/google-native/0.27.0.0/index.html
+[google-native-registry-docs]: https://www.pulumi.com/registry/packages/google-native
+[pulumi-registry]: https://www.pulumi.com/registry
+[issues-pulumi-kotlin]: https://github.com/VirtuslabRnD/pulumi-kotlin/issues
+[support-for-idiomatic-kotlin-issue]: https://github.com/pulumi/pulumi-java/issues/544
+[pulumi-slack-java-channel]: https://pulumi-community.slack.com/archives/C03DPAY96NB
+[calendly-feedback-meeting]: https://calendly.com/michalfudala/kotlin-sdk-for-pulumi-feedback
+
```

Comment:
(the most minor comment in the world)

I'd remove the double extra line at the end of file :D

Btw, there's an IntelliJ setting for that:
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/34098778/212123460-2b142a35-160e-49e6-8253-4292eb7753fa.png">
Unfortunately, it's not configurable through the formatter